c10/cuda/CUDACachingAllocator.cpp

```diff
@@ -843,8 +843,7 @@ struct AllocParams {
       size_t size,
       cudaStream_t stream,
       BlockPool* pool,
-      size_t alloc_size,
-      DeviceStats& stats)
+      size_t alloc_size)
```

Comment:
Is this part of our public API? How did you check there are no usage if it in the wild?

Reply:
It is defined in cpp file and not exposed to be public.